core/src/Streamly/Internal/Data/Serialize/TH.hs

```diff
@@ -10,6 +10,8 @@
 --
 module Streamly.Internal.Data.Serialize.TH
     ( deriveSerialize
+    , SerializeTHConfig(..)
+    , defaultSerializeTHConfig
```

Comment:
I would prefer to call these just `Config` and `defaultConfig`?

In fact we can assume this module to be imported qualified and drop the `Serialize` suffix from `deriveSerialize` as well.

Reply:
Gotcha.